### ISSUE TITLE
No Monster Data

### ISSUE THREAD

**Issue by Snitzle** [LartTyler/MHWDB-Docs#31]
It would be fantastic to be able to pull through data on monsters. Any chance this is in the pipeline?

**Comment by LartTyler** [comment]
It's something that I'd like to add, I just haven't had much time lately to go in and add new features. I might have some free time this weekend that I can look into getting some basic information added. Were you just looking for things like elemental resistances and possible drops?

**Comment by Snitzle** [comment]
Types, Weaknesses, Item drops that sort of thing. Currently learning JS and would like to use this for getting my first app together. Does help I'm in love with MH and would enjoy using my own tool too aha.

**Comment by yoonhok524** [comment]
Oh... it is already requested...
LartTyler, I appreciate your good data.
If the monster data is added, it may be the Best of the API.
How does it going on?

**Comment by LartTyler** [comment]
It's definitely planned, but I've kinda put new features on hold for the time being, until I have more time to commit to the project. Once I get a chance to start working on it, I'll let you know!

**Comment by Snitzle** [comment]
I appreciate the effort, thanks!


On 30 Jul 2018 at 15:53, <Tyler Lartonoix <notifications@github.com>>
wrote:

It's definitely planned, but I've kinda put new features on hold for the
time being, until I have more time to commit to the project. Once I get a
chance to start working on it, I'll let you know!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/LartTyler/MHWDB-Docs/issues/31#issuecomment-408891693>,
or mute the thread

**Comment by yoonhok524** [comment]
Thanks to reply.
You know, MHW PC version will be launched soon.
So I want to use the Monster API before it. T.T
Anyway I appreciate your effort :)

**Comment by Acantore** [comment]
Well hello there!
I was wondering if there is any updates on this topic.
Love your API and docs! Keep up the good work!

Greetings from germany by the way.

**Comment by LartTyler** [comment]
@Acantore As part of [this pull request](https://github.com/LartTyler/MHWDB-API/pull/79), I've added support for monster data (as well as a few other things), but that release is currently in limbo while I work on moving the API off of a scraper based system to collect it's data to something that I (and select contributors) can manage by hand. I haven't had as much time as I'd like to devote to it, but my hope is to get it wrapped up in the next few weeks.

If you're interested in following (or contributing to) that part of the project, you can find the frontend app's repository [here](https://github.com/LartTyler/MHWDB-Contrib). The backend code is finished and part of that PR I linked before, and once the frontend is finished, I'll go in and enter all the monster data.

**Comment by Acantore** [comment]
Thank you for your kind reply.
I will keep an eye open and watch the repository, maybe look into contributing after i finished my WebApp.

Keep up the good work!

**Comment by LartTyler** [comment]
Monster support has been added with [v1.14.0](https://github.com/LartTyler/MHWDB-Docs/releases/tag/1.14.0). Since the information now needs to be manually entered, I'll be doing that over the next few days, and will update this issue once everything is in there. If you guys are still interested in this and would like to help with some data entry, let me know!
